karma.conf.js

```diff
@@ -0,0 +1,52 @@
+const path = require('path')
+const webpackConfig = require('./webpack.config.js')
+
+const IS_TRAVIS = process.env.TRAVIS
+
+module.exports = function(config) {
+  config.set({
+    singleRun: true,
+    browsers: IS_TRAVIS ? ['HeadlessChrome'] : ['Chrome'],
+    frameworks: ['mocha', 'chai'],
+
+    files: [
+      path.join(__dirname, 'client', '**', '*.spec.js'),
+    ],
+
+    reporters: ['dots'],
+
+    preprocessors: {
+      'client/**/*.spec.js': ['webpack'],
+    },
+
+    webpack: {
+      mode: 'development',
+      devtool: webpackConfig.devtool,
+      module: webpackConfig.module,
+    },
```

Comment:
the `karma`/`webpack` integration needs some configuration. Here `webpackConfig` is the configuration from `./webpack.config.js`. We don't need the entire configuration as we won't be writing the output to disk.